File: Python Fundamentals/Functions/odd_and_even_sum.py
def sum_even_odd(value):
    sum_even = 0
    sum_odd = 0

    for i, symbol in enumerate(value):
        digit = int(symbol)

        if digit % 2 == 0:
            sum_even += digit
        elif digit % 2 == 1:
            sum_odd += digit

    return f"Odd sum = {sum_odd}, Even sum = {sum_even}"


number = input()
result = sum_even_odd(number)
print(result)

File: JavaScript Fundamentals/Functions/oddAndEvenSum.js
function solve(num) {
    let number = String(num);
    let evenSum = 0;
    let oddSum = 0;

    for (let index = 0; index < number.length; index++) {
        Number(number[index]) % 2 == 0 ? evenSum += Number(number[index]) : oddSum += Number(number[index]);
    }

    console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`);
}
